### DISCUSSION TITLE
Lazygit doesn't show the file edit changes in commit

### DISCUSSION THREAD

**Discussion by halfpastfive23** [jesseduffield/lazygit#3964] [Q&A] [answered]
I was following a tutorial on how to use lazygit and I was stuck at when I made the changes in a file it doesn't show the current changed or edited file in commit.

**Comment by mark2185** [comment]
What tutorial was it? Could you please describe the steps you took?

**Reply by halfpastfive23** [reply]
https://www.youtube.com/watch?v=Ihg37znaiBo&t=129s&ab_channel=JoseanMartinez 

This is the video and u can watch it starting at 2:20 where he edit a file and it shows the edit been made in lazygit

**Reply by mark2185** [reply]
Yes, and that's the expected behavior.

Could you please describe the steps you did? If you see a change in `git status`, you should see it in `lazygit`.

**Reply by halfpastfive23** [reply]
I created a repository but I didn't add those files in.. so it probably won't show the changes been made to a specific files unless I add them in the repository right. If that's true than its on me as I didn't add the files beforehand to see the changes been made from the lazygit.

**Reply by mark2185** [reply, marked as answer]
The files should appear as "untracked", same as they would when you run `git status`, unless you have explicitly disabled that in your `gitconfig`.
![image](https://github.com/user-attachments/assets/bbdeabc0-02f9-4a08-8b21-d29a70924f63)

But yes, in order to see the diff from a previous state of a file, you need to add/commit them.

**Reply by halfpastfive23** [reply]
Appreciated for the info.
